.github/workflows/cd-manager.yml

```diff
@@ -0,0 +1,121 @@
+      -
+        name: Set rc branch tag
+        if: ${{ env.BRANCH == 'release-candidate' }}
+        run: |
+          echo "ENV_TAG=tnet" >> $GITHUB_ENV
+      -
+        name: Set develop branch tag
+        if: ${{ env.BRANCH == 'develop' || env.ENV_TAG == '' }}
+        run: |
+          echo "ENV_TAG=dev" >> $GITHUB_ENV
+      -
+        name: Set commit status "pending"
+        run: |
+          res=$(curl \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ env.GH_TOKEN }}" \
+            https://api.github.com/repos/3box/pipeline-tools/statuses/${{ env.SHA }} \
+            -d '{"state":"pending","target_url":"${{ env.RUN_URL }}","description":"Image verification started","context":"ci/image: verify"}') || exit 1
+          echo $res
+          if [[ $res != *"pending"* ]]; then
+            exit 1
+          fi
+      -
+        name: Install Dagger
+        uses: dagger/dagger-for-github@v3
+        with:
+          install-only: true
+      -
+        name: Setup Dagger
+        run: |
+            dagger project init
+            dagger project update
+            dagger project update "github.com/3box/pipeline-tools@v${{ env.PIPELINE_TOOLS_VER }}"
+
+            AWS_SCRIPTS=cue.mod/pkg/universe.dagger.io/aws/_scripts
+            mkdir -p $AWS_SCRIPTS
+            curl -fsSL "https://raw.githubusercontent.com/dagger/dagger/v${{ env.DAGGER_VERSION }}/pkg/universe.dagger.io/aws/_scripts/install.sh" > $AWS_SCRIPTS/install.sh
+            chmod +x $AWS_SCRIPTS/install.sh
+      -
+        name: Verify Docker image
+        run: |
+          cd cd/manager
+          dagger do verify -p ${{ env.DAGGER_PLAN }}
+      -
+        name: Push Docker image
+        if: ${{ env.BRANCH == 'main' || env.BRANCH == 'release-candidate' || env.BRANCH == 'develop' }}
+        run: dagger do push -w "actions:push:\"${{ env.AWS_REGION }}\":\"${{ env.ENV_TAG }}\":\"${{ env.BRANCH }}\":\"${{ env.SHA }}\":\"${{ env.SHA_TAG }}\":_" -p ${{ env.DAGGER_PLAN }}
+      -
+        name: Set commit status "success"
+        run: |
+          res=$(curl \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ env.GH_TOKEN }}" \
+            https://api.github.com/repos/3box/pipeline-tools/statuses/${{ env.SHA }} \
+            -d '{"state":"success","target_url":"${{ env.RUN_URL }}","description":"Image verification successful","context":"ci/image: verify"}') || exit 1
+          echo $res
+          if [[ $res != *"success"* ]]; then
+            exit 1
+          fi
+      -
+        name: Set commit status "failure"
+        if: ${{ failure() }}
+        run: |
+          res=$(curl \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ env.GH_TOKEN }}" \
+            https://api.github.com/repos/3box/pipeline-tools/statuses/${{ env.SHA }} \
```

Comment:
do we have the ability to make variables for some of these long paths?

Reply:
Yes, I can do that.